l2geth/cmd/devp2p/dns_cloudflare.go

```diff
@@ -79,7 +80,7 @@ func (c *cloudflareClient) checkZone(name string) error {
 		c.zoneID = id
 	}
 	log.Info(fmt.Sprintf("Checking Permissions on zone %s", c.zoneID))
-	zone, err := c.ZoneDetails(c.zoneID)
+	zone, err := c.ZoneDetails(context.Background(), c.zoneID)
```

Comment:
Was this causing build issues?

Reply:
yes, pulling in secp256k1 from geth into l2geth caused this dependency to update. unfortunately the newer version had an incompatible API, but the changes were pretty minimal